bigchaindb/pipelines/vote.py

```diff
@@ -0,0 +1,138 @@
+"""This module takes care of all the logic related to block voting.
+
+The logic is encapsulated in the ``Vote`` class, while the sequence
+of actions to do on transactions is specified in the ``create_pipeline``
+function.
+"""
+
+from collections import Counter
+
+from multipipes import Pipeline, Node
+
+from bigchaindb.pipelines.utils import ChangeFeed
+from bigchaindb import Bigchain
+
+
+class Vote:
+    """This class encapsulates the logic to vote on blocks.
+
+    Note:
+        Methods of this class will be executed in different processes.
+    """
+
+    def __init__(self):
+        """Initialize the Block voter."""
+
+        last_voted = Bigchain().get_last_voted_block()
```

Comment:
It's to overcome a limitation with `multiprocessing`. Will add a comment to explain that :)